cmd/system-probe/modules/network_tracer.go

```diff
@@ -110,6 +110,8 @@ func (nt *networkTracer) Register(httpMux *module.Router) error {
 		}
 		count := runCounter.Inc()
 		logRequests(id, count, len(cs.Conns), start)
+
+		nt.tracer.ReleaseProtocolStats(cs)
```

Comment:
why is it here? why do we need to expose a method in the NPM tracer to release USM stats?